GameOfLifeApp/GameLogic.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLifeApp
+{
+    public class GameLogic
+    {
+        bool[,] generatedField;
+
+        /// <summary>
+        /// Holds user data of array size.    
+        /// </summary>
+        public int countOfRows { get; set; }
```

Comment:
Public class properties should be in PascalCase - `CountOfRows`.

But, think if you need such properties, since this information can be access already from the bool[,] field arrays

Reply:
Done. PascalCase for class names.